### DISCUSSION TITLE
Error trapped ID 760 for have changed shell element's reference part

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1516] [Q&A] [answered]
Hi, I'm trying to change some element's reference part with ls prepost and then run a simulation with openradioss.
I cannot perform the simulation cause an error trapped (ID 760).
I tried the same procedure with a model with lower elements and i didn't have any problems.
What could be the problem?
In the attachment the .out file.
[extruded_avff_0000.out.txt](https://github.com/OpenRadioss/OpenRadioss/files/11592534/extruded_avff_0000.out.txt)

**Comment by FGarelli** [comment]
I noticed that is contact problem cause when i cancel that card the error disappear. I'm currently using *CONTACT_AUTOMATIC_SINGLE_SURFACE.
The starter file is the following.
[extruded.k.txt](https://github.com/OpenRadioss/OpenRadioss/files/11599832/extruded.k.txt)
What I'm trying to do is changing these blue elements properties 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/14d9699e-4ffe-4639-8f2e-ef418a42b90a)

**Comment by laurent-altr** [comment]
Hello @FGarelli, 

It seems to be a bug. I have created the issue: https://github.com/OpenRadioss/OpenRadioss/issues/1517
Thank you for reporting this!

**Comment by Schwoertzig** [comment]
Hi @FGarelli , could you please share the input file you obtained from "extruded.k.txt" when you changed the properties ? I mean the input file which lead to the error.

You may also 
1. compile a debug version :
    (linux) ./build_script.sh -arch=linux64_gf -debug=1
2. launch this debug version with -notrap option :
     (linux) starter_linux64_gf -i extruded.k.txt **-notrap**
you should then be able to locate the issue from the backtrace. 

Regards

**Reply by FGarelli** [reply]
Hi @Schwoertzig , you can find the file in the main discussion's post.
Here the error:

 INTERFACE NUMBER. . . . . . . . . . . . . .         1
 INTERFACE TYPE. . . . . . . . . . . . . . .    25

 +=============================================================+
 |     ABNORMAL END                                            |
 |          Segmentation Violation                             |
 |     TRACE BACK :                                            |
 +=============================================================+
    INTERFACES                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      
 +=============================================================+

ERROR ID :    760
** ERROR TRAPPED


 RADIOSS STOP DUE TO INPUT ERROR

I'm using a windows version, what are the command line for that?

**Reply by Schwoertzig** [reply, marked as answer]
Hi @FGarelli , I did not reproduce on Linux. 
 It is probably an issue which is specific to Windows. I am going to check with a Windows user.
( for your information "-notrap" should also work when launching Starter on Windows)
 Regards

**Reply by FGarelli** [reply]
Hi, inserting the -notrap option it worked!

**Comment by qiangRadioss** [comment]
i didn't reproduce the issue on windows (both win64, win64_db)
[extruded_0000.out.txt](https://github.com/OpenRadioss/OpenRadioss/files/11611812/extruded_0000.out.txt)
regards.

**Comment by MarianBulla** [comment]
On my side, it went also well, until the end. And ended with NORMAL TERMINATION (see attached files).
I used windows 11 and a recent compiled OpenRadioss (from 7th Aug. 2023).
[extruded.zip](https://github.com/OpenRadioss/OpenRadioss/files/12303034/extruded.zip)

Can you retry it with a newer OpenRadioss version, please ?

Thanks and best regards,
Marian

**Comment by Sanketh-R** [comment]
Dear Users
I'm a beginner to the Openradioss. I tried to follow the steps in the YouTube video - "How to use OpenRadioss - Quick demo (windows 11)". When I tried to submit the example model from the website, I got the following error:

WARNING ID :   1005
** WARNING KMP_STACKSIZE ENVIRONMENT VARIABLE IS SET TO LESS THAN 380M

 .. FUNCTIONS & TABLES

 .. MATERIALS                                                                            

 .. NODES                                                                                

 ** ERROR TRAPPED : REPORT TO LISTING FILE

ERROR ID :    760

I'm using Windows 11 and a recently compiled OpenRadioss (12th Jan 2024). The above error might be pertinent to Windows 11.
Kindly provide some insights on how to overcome the error.

Thanks and best Regards
Sanketh

**Reply by OlivierWienholtz** [reply]
Hi  Sanketh,
Just wat to know what compiler you have used and 
I'l be glad if you can tell which Model you tried ? 

Thank you
Olivier
